website/content/docs/nia/configuration.mdx

```diff
@@ -40,6 +48,13 @@ buffer_period {
   - `facility` - (string: "local0") Name of the syslog facility to log to.
   - `name` - (string: "consul-terraform-sync") Name to use for the daemon process when logging to syslog.
 - `working_dir` - (string: "sync-tasks") Specifies the base working directory for managing the artifacts generated by Consul-Terraform-Sync for each task, including Terraform configuration files. Consul-Terraform-Sync will also generate a subdirectory for each task, e.g., `./sync-tasks/task-name`. The subdirectory is the working directory for the task. Use the [`task.working_dir`](#working_dir-1) option to override task-level working directories.
+- `tls` - Configure TLS on the Consul-Terraform-Sync API.
```

Comment:
We'll eventually want to figure out how to structure this so that we can implement tables as necessary for this kind of information. I think that doing so is a heavier lift than we want to undertake for this PR, though.